schema/fs.go

```diff
@@ -379,735 +452,9107 @@ hR+PkX5zgcX30ucJvpgCXqksmwP73wn8U2qZo81xwnEy+6yyZ6Vexsl79az+DgAA//850zAaggUAAA==
 
 	"/fs.go": {
```

Comment:
And in case it helps, here's the history of the recursive `/fs.go`'s presence in master:

```
$ git log --decorate -p -G '/fs.go' origin/master -- schema/fs.go | grep '^commit \|Author\|"/fs.go"'
commit 0556a6bee5ee1514762de05fd95d7af09b38f07f (origin/pr/533)
Author: Vincent Batts <vbatts@hashbangbash.com>
+       "/fs.go": {
commit 10e6ff7101c2b0ff76e41a4a213838953e2082d8 (origin/pr/78)
Author: Jonathan Boulle <jonathanboulle@gmail.com>
-       "/fs.go": {
commit 4263cf529b30d46b99a761ca764ede70b667cdc9 (origin/pr/65)
Author: Sergiusz Urbaniak <sur@coreos.com>
+       "/fs.go": {
```

Since #533, only @xiekeyang has rebuilt `fs.go`:

```
$ git log --decorate 0556a6be..origin/master -- schema/fs.go | grep '^commit \|Author'
commit 5bf90dd1ed7a0607c7fdc6013e3841952d228d43 (origin/pr/564)
Author: xiekeyang <xiekeyang@huawei.com>
```

Who seems to have the same buggy `esc` behavior:

```
$ git cat-file -p 5bf90dd1:schema/gen.go | grep 'generate esc'
//go:generate esc -private -pkg=schema -ignore=.*go -ignore=.*swp .
$ git cat-file -p 5bf90dd1:schema/fs.go | grep '"/'
        "/config-schema.json": {
        "/config_test.go": {
```

My theory is still that you have an old, buggy copy of `esc` ;).